CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-22.10.1
+
+This edge release fixes some sections of the Viz dashboard that were appearing
+blank, and adds an optional PodMonitor resource to the Helm chart which allows
+having an easier integration with the Prometheus Operator. It also includes many
+fixes submitted by our contributors.
+
+* Fixed the dashboard sections Tap, Top, Routes that were appearing blank
+  (thanks @MoSattler!)
+* Added an optional PodMonitor resource to the main Helm chart (thanks
+  @jaygridley!)
+* Fixed the CLI's `--api-addr` flag (thanks @mikutas!)
+* Expanded the `linkerd authz` command to display AuthorizationPolicy resources
+  that target namespaces (thanks @aatarasoff!)
+* Fixed interpretation of the `NotIn` label selector operator in the policy
+  resources
```

Comment:
take it or leave it: "fixed interpretation"  feels a little vague to me, maybe we should note specifically that it was essentially being treated as an `In` operator previously?